openvpn/process.go

```diff
@@ -24,32 +24,42 @@ import (
 
 	"github.com/mysteriumnetwork/node/openvpn/config"
 	"github.com/mysteriumnetwork/node/openvpn/management"
+	"github.com/mysteriumnetwork/node/openvpn/tunnel"
 )
 
-// Process defines openvpn process interface with basic controls
-type Process interface {
-	Start() error
-	Wait() error
-	Stop()
+type tunnelSetup interface {
+	Setup(config *config.GenericConfig) error
+	Teardown()
 }
 
 type openvpnProcess struct {
```

Comment:
As far as I remember OpenvpnProcess is part of public api ( it's returned from package to outside world) and can be safely made public. As general approach - structures returned by New.. functions should always be exported ones. Even linter mentions that.

Reply:
👍 I think the same